static/js/ProfilePage.js

```diff
@@ -15,10 +15,19 @@ document.addEventListener('DOMContentLoaded', function(){
         document.querySelector('.overlay').style.display = 'block';
     });
 
-    let closeButton = document.getElementById("close_button");
+    //These buttons are for the close and submit buttons.
```

Comment:
This code should be inside the DomContentLoaded function to prevent being called before the dom is loaded

Reply:
They are all inside the DOMContentLoaded function